src/app/features/ahbs/components/format-version-select/format-version-select.component.html

```diff
@@ -1,11 +1,20 @@
 @if (formatVersions$ | async; as formatVersions) {
-  <select
-    [formControl]="control"
-    (change)="onChange ? onChange(control.value) : null"
-  >
-    <option value="">Bitte auswählen</option>
-    @for (formatVersion of formatVersions; track formatVersion) {
-      <option [value]="formatVersion">{{ formatVersion }}</option>
-    }
-  </select>
+  <div class="flex flex-col items-start mt-2 h-auto w-full">
+    <select
+      id="format-version-select"
+      class="inline-block border-2 border-white rounded bg-[#F6ECED] py-3 ps-3 pe-24 focus:outline-0 w-full"
```

Comment:
bg-[#F6ECED] is this a background color?
Is it possible to define colours in tailwind globally?

Reply:
Yes sure, once we have the full design we should define more stuff